src/content/tools/sdk.md

```diff
@@ -41,3 +41,10 @@ The [`dart` CLI tool][] is available with the Flutter SDK at `flutter/bin/dart`.
 [`flutter` CLI tool]: /reference/flutter-cli
 [Install]: /get-started/install
 [README file]: {{site.repo.flutter}}/blob/main/README.md
+
+## SDK support in tooling
```

Comment:
Maybe "SDK support for Flutter developer tools" ?

Reply:
thanks! done